selector-set.next.js

```diff
@@ -310,6 +310,7 @@ SelectorSet.prototype.remove = function(selector, data) {
             if (obj.selector === selector && (removeAll || obj.data === data)) {
               objs.splice(k, 1);
               removedIds[obj.id] = true;
+              removedSelectors[obj.selector] = true;
```

Comment:
I think removing by selector string might not work when two selectors exist with different data properties.

```js
set.add('.foo', 1)
set.add('.foo', 2)
set.remove('.foo', 2)
set.selectors // should still include '.foo'
set.remove('.foo', 1)
set.selectors // should now be empty
```

Reply:
Add a test case that passes on `master` but fails on this branch 2445317a005833f1e39104eac1f30a1eb61c1a4d